db/index.js

```diff
@@ -14,8 +14,20 @@ let sellerSchema = mongoose.Schema({
 let Seller = mongoose.model('Seller', sellerSchema);
 
 
+let getSellerData = (seller, callback) => {
+
+  Seller.findOne({ seller }, (err, results) => {
```

Comment:
This is where you might want to do look up by id rather than name. In an app this small it doesn't matter too much but for the future when things are larger scale and you may have duplicate names I think its best practice to do look ups by id if possible